### DISCUSSION TITLE
Welcome to tamilblasters_stremio_addon Discussions!

### DISCUSSION THREAD

**Discussion by mhdzumair** [mhdzumair/tamilblasters_stremio_addon#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jrjaggari** [comment]
Installed version 2.0 and I am getting the following error, "API Exception occured, Please try again or Submit an issue in github repository tamilblasters_stremio_addon" when I try to play a video.

**Comment by nitzz204u** [comment]
It's not working in TV.
